integration/integrate-with-amazon-s3.adoc

```diff
@@ -0,0 +1,30 @@
+[id="integrate-with-amazon-s3"]
+= Integrating with Amazon S3
+include::modules/common-attributes.adoc[]
+:context: integrate-with-amazon-s3
+
+toc::[]
+
+[role="_abstract"]
+
+You can integrate {product-title} with https://aws.amazon.com/s3/[Amazon S3] to enable data backups.
+You can use these backups for data restoration in the case of an infrastructure disaster, or corrupt data.
+After you integrate with Amazon S3, you can schedule daily or weekly backups and do manual on-demand backups.
+
+The backup includes {product-title}'s entire database, which includes all configurations, resources, events, and certificates.
+Make sure that backups are stored securely.
+
+[IMPORTANT]
+====
+* If you are using {product-title} version 3.0.53 or older, the backup does not include certificates.
+* If your Amazon S3 is part of an air-gapped environment, you must add your AWS root CA as a xref:../configuration/add-trusted-ca.adoc[trusted certificate authority] in {product-title}.
+====
+
+include::modules/amazon-s3-configuring-acs.adoc[leveloffset=+1]
+
+include::modules/perform-on-demand-backups-amazon-s3.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Just checking, is this additional resources section supposed to be for the overall assembly, or smushed with the above module?

If it's for the overall assembly, then it should use the `== Additional resources` format (with an anchor).